config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -508,3 +508,28 @@ periodics:
       - ../kubernetes/vendor/k8s.io/apimachinery/...
       - ../kubernetes/vendor/k8s.io/apiserver/...
       - ../kubernetes/vendor/k8s.io/client-go/...
+
+- name: ci-build-and-push-k8s-at-golang-tip
```

Comment:
This file is huge already, let's create a separate one for golang jobs, e.g. sig-scalability-golang.yaml

Reply:
Done.